scripts/gas-refund-program/transactions-indexing/swaps-subgraph.ts

```diff
@@ -87,14 +93,23 @@ export async function getSuccessfulSwaps({
     skip = skip + pageSize;
   }
 
-  return swaps;
+  // allow dupes until epoch 11, after which throw an error
+  if (epoch < GasRefundDeduplicationStartEpoch) {
+    return swaps
+  } else {
+    const uniqSwaps = [...new Set(swaps.map(swap => swap.txHash))]
```

Comment:
gotta filter swaps based on txOrigin === initiator before doing uniqueness check

Reply:
early return while you're at it 🙏